client/keys/add.go

```diff
@@ -200,6 +195,11 @@ func runAddCmd(_ *cobra.Command, args []string) error {
 		if err != nil {
 			return err
 		}
+
+		if !bip39.IsMnemonicValid(mnemonic) {
+			_, _ = fmt.Fprintf(os.Stderr, "Error: Mnemonic is not valid.\n")
```

Comment:
```suggestion
			_, _ = fmt.Fprintf(os.Stderr, "Error: invalid or empty mnemonic\n")
```

Reply:
Empty mnemonic is a kind of invalid mnemonic. :)
```go
                if !bip39.IsMnemonicValid(mnemonic) {
			return errors.New("invalid mnemonic")
		}
```